curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/664ed97c55a99e5ffab759dc.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 664ed97c55a99e5ffab759dc
+title: Step 12
+challengeType: 0
+dashedName: step-12
+---
+
+# --description--
+
+Now it is time to review the concepts you have learned so far. This review step is to ensure you have a good grasp on the basics so you can apply them to the pyramid generator project later on. 
+
+In this step you will need to do the following:
+
+- Use the `let` keyword to declare a variable called `profession` and assign it the string `"teacher"`.
+- Use the `let` keyword to declare a variable called `age`. **Do not** assign it a value.
+- Log to the console the value of the `age` variable. You should see `undefined` in the console.
+- Log to the console the value of the `profession` variable. You should see `"teacher"` in the console.
+
+# --hints--
+
+You should declare a variable called `profession`.
+
+```js
+assert.match(code, /profession/);
+```
+
+You should use `let` to declare the `profession` variable.
+
+```js
+assert.match(code, /let\s+profession/);
+```
+
+You should assign the string `"teacher"` to the `profession` variable.
+
+```js
+assert.equal(profession, "teacher");
+```
+
+You should have a variable called `age`
+
+```js
+assert.match(code, /age/);
```

Comment:
```suggestion
assert.match(code, /(?:var|let|const)\s*=\s*age/);
```

Reply:
The same is has been done for the age variable